Makefile

```diff
@@ -263,3 +265,13 @@ test: .env demo
 
 wipe: .env
 	@./scripts/renku-wipe.sh
+
+.PHONY: minikube-deploy
+minikube-deploy:
```

Comment:
shouldn't this depend on `clone`? i.e. clone a repo if it isn't there?

Reply:
true